masq/src/commands/set_exit_location_command.rs

```diff
@@ -0,0 +1,224 @@
+        masq> exit-location --country-codes \"CZ|SK\"                             // fallback-routing is OFF, \"CZ\" and \"SK\" countries has different prirority\n";
+
+// TODO update following help when GH-469 is done
+const COUNTRY_CODES_HELP: &str = "Establish a set of countries that your Node should try to use for exit Nodes. You should choose from the countries that host the \
+        Nodes in your Neighborhood. List the countries in order of preference, separated by vertical pipes (|). If your level of preference \
+        for a group of countries is the same, separate those countries by commas (,).\n\
+        To obtain codes you cant use 'country-codes-list' command. You can specify country codes followingly:\n\n\
+        masq> exit-location --country-codes \"CZ,PL|SK\"                          // \"CZ\" and \"PL\" countries has same priority \"1\", \"SK\" has prirority \"2\" \n\
+        masq> exit-location --country-codes \"CZ|SK\"                             // \"CZ\" and \"SK\" countries has different prirority\n";
+
+const FALLBACK_ROUTING_HELP: &str = "If you just want to make a suggestion, and you don't mind Nodes in other countries being used if nothing is available \
+     in your preferred countries, you can specify --fallback-routing, and you'll get no error unless there are no exit Nodes \
+     available anywhere. \n Here are some examples: \n\n\
+     masq> exit-location --country-codes \"CZ\" --fallback-routing              // Set exit-location for \"CZ\" country with fallback-routing on \n\
+     masq> exit-location --country-codes \"CZ\"                                 // Set exit-location for \"CZ\" country with fallback-routing off \n";
+
+pub fn set_exit_location_subcommand() -> App<'static, 'static> {
+    SubCommand::with_name("exit-location")
+        .about(EXIT_LOCATION_ABOUT)
+        .arg(
+            Arg::with_name("country-codes")
+                .long("country-codes")
+                .value_name("COUNTRY-CODES")
+                .validator(common_validators::validate_exit_locations)
+                .help(COUNTRY_CODES_HELP)
+                .required(false),
+        )
+        .arg(
+            Arg::with_name("fallback-routing")
+                .long("fallback-routing")
+                .value_name("FALLBACK-ROUTING")
+                .help(FALLBACK_ROUTING_HELP)
+                .takes_value(false)
+                .required(false),
+        )
+}
+
+pub mod tests {
+    use crate::commands::commands_common::{Command, STANDARD_COMMAND_TIMEOUT_MILLIS};
+    use crate::commands::set_exit_location_command::SetExitLocationCommand;
+    use crate::test_utils::mocks::CommandContextMock;
+    use masq_lib::messages::{
+        CountryCodes, ToMessageBody, UiSetExitLocationRequest, UiSetExitLocationResponse,
+    };
+    use std::sync::{Arc, Mutex};
+
+    #[test]
+    fn can_deserialize_ui_set_exit_location() {
+        let expected_request = UiSetExitLocationRequest {
```

Comment:
If this variable isn't used anywhere in Act or Arrange, could you move it down to Assert? It's part of the assertion logic, and having it up here in Act is confusing. It may sometimes be necessary, but if it's not, the confusion can be avoided.